iothub/device/src/Exceptions/IotHubClientException.cs

```diff
@@ -140,6 +140,7 @@ internal IotHubClientException(IotHubClientErrorCode errorCode) : base()
         /// </summary>
         /// <param name="info">The <see cref="SerializationInfo"/> that holds the serialized object data about the exception being thrown.</param>
         /// <param name="context">The <see cref="StreamingContext"/> that contains contextual information about the source or destination.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public override void GetObjectData(SerializationInfo info, StreamingContext context)
```

Comment:
Do we even need this ctor? Last I checked, it was unreferenced

Reply:
Methods and ctors for de/serialization won't be referenced, but they are called at runtime.

I don't know the point of this `GetObjectData` method, though.